modules/ROOT/pages/mongodb/mongodb-connector-reference.adoc

```diff
@@ -14,2281 +18,2415 @@ MongoDB is an open source, high-performance, schema-free, document-oriented data
 +++
 
-=== Parameters
-
-[cols=".^20%,.^20%,.^35%,.^20%,^.^5%", options="header"]
+==== Parameters
+[%header%autowidth.spread]
 |===
 | Name | Type | Description | Default Value | Required
 | Configuration | String | The name of the configuration to use. | | x
-| Find Query a| Binary |  +++the String mandatory query+++ |  | x
-| Target Variable a| String |  +++The name of a variable on which the operation's output will be placed+++ |  |
-| Target Value a| String |  +++An expression that will be evaluated against the operation's output and the outcome of that expression will be stored in the target variable+++ |  +++#[payload]+++ |
+| Find Query a| Binary |  +++The String mandatory query+++ |  | x
+| Target Variable a| String |  +++The name of a variable to store the operation's output.+++ |  | 
+| Target Value a| String |  +++An expression to evaluate against the operation's output and store the expression outcome in the target variable+++ a|  `#[payload]` | 
 | Reconnection Strategy a| * <<reconnect>>
-* <<reconnect-forever>> |  +++A retry strategy in case of connectivity errors+++ |  |
+* <<reconnect-forever>> |  +++A retry strategy in case of connectivity errors.+++ |  | 
 |===
 
-=== Output
-
-[cols=".^50%,.^50%"]
+==== Output
+[%autowidth.spread]
 |===
-| Type a| Object
+|Type |Object
 |===
 
 === For Configurations
-
-* <<config>>
-
-=== Throws
-
-* MONGO:WRITE_CONCERN
-* MONGO:RETRY_EXHAUSTED
-* MONGO:UNKNOWN
-* MONGO:SERVER_ERROR
-* MONGO:BULK_WRITE
-* MONGO:IO_EXCEPTION
-* MONGO:SOCKET_READ_ERROR
-* MONGO:WRITE
-* MONGO:CLIENT_EXCEPTION
-* MONGO:QUERY_ERROR
-* MONGO:NODE_IS_RECOVERING
-* MONGO:MONGO_GRID_FS
-* MONGO:INVALID_CREDENTIALS
-* MONGO:EXECUTION_TIMEOUT
-* MONGO:SOCKET
-* MONGO:COMMAND_ERROR
-* MONGO:MongoExecution
-* MONGO:SOCKET_CLOSED
-* MONGO:SECURITY
-* MONGO:WAIT_QUEUE_FULL
-* MONGO:CURSOR_NOT_FOUND
-* MONGO:SOCKET_OPEN
-* MONGO:INTERNAL_EXCEPTION
-* MONGO:INTERRUPTED
-* MONGO:SOCKET_WRITE_ERROR
-* MONGO:CONNECTIVITY
-* MONGO:NOT_PRIMARY
-* MONGO:TIMEOUT
-* MONGO:DUPLICATE_KEY
-* MONGO:INCOMPATIBLE_DRIVER
-* MONGO:SOCKET_READ_TIMEOUT
-* MONGO:MAPPING_EXCEPTION
-* MONGO:ILLEGAL_ARGUMENT
+* <<config>> 
+
+==== Throws
+* MONGO:RETRY_EXHAUSTED 
+* MONGO:TIMEOUT 
+* MONGO:WAIT_QUEUE_FULL 
+* MONGO:COMMAND_ERROR 
+* MONGO:SOCKET 
+* MONGO:SOCKET_WRITE_ERROR 
+* MONGO:SOCKET_READ_TIMEOUT 
+* MONGO:INTERRUPTED 
+* MONGO:NOT_PRIMARY 
+* MONGO:WRITE 
+* MONGO:SOCKET_CLOSED 
+* MONGO:UNKNOWN 
+* MONGO:CURSOR_NOT_FOUND 
+* MONGO:INVALID_CREDENTIALS 
+* MONGO:ILLEGAL_ARGUMENT 
+* MONGO:WRITE_CONCERN 
+* MONGO:CLIENT_EXCEPTION 
+* MONGO:IO_EXCEPTION 
+* MONGO:SOCKET_READ_ERROR 
+* MONGO:QUERY_ERROR 
+* MONGO:MONGO_GRID_FS 
+* MONGO:SOCKET_OPEN 
+* MONGO:CONNECTIVITY 
+* MONGO:EXECUTION_TIMEOUT 
+* MONGO:SECURITY 
+* MONGO:MAPPING_EXCEPTION 
+* MONGO:NODE_IS_RECOVERING 
+* MONGO:INTERNAL_EXCEPTION 
+* MONGO:SERVER_ERROR 
+* MONGO:DUPLI
```

Comment:
I'm wondering if the formatting here is correct. Maybe it should be something like this?
Example: `<$set,{"key":123}>`

Reply:
Changed line 2156 to:
For example: `<$set,{"key":123}>`